src/transformers/pipelines/text2text_generation.py

```diff
@@ -159,13 +159,16 @@ def postprocess(self, model_outputs, return_type=ReturnType.TEXT, clean_up_token
         if return_type == ReturnType.TENSORS:
             record = {f"{self.return_name}_token_ids": model_outputs}
```

Comment:
This will break `ReturnType.TENSORS` since now the return types would be different.

We can use 

```python
for sequence in model_outputs["output_ids"]:
   if return_type == ReturnType.TENSORS:
      ... old code
   elif return_type = ....:
     ... old _code
   records.append(record)
return records
```

Reply:
Thanks for your comments. I modified the corresponding code